gRPC-Core.podspec

```diff
@@ -185,8 +185,8 @@ Pod::Spec.new do |s|
     ss.header_mappings_dir = '.'
     ss.libraries = 'z'
     ss.dependency "#{s.name}/Interface", version
-    ss.dependency 'BoringSSL-GRPC', '0.0.5'
-    ss.compiler_flags = '-DGRPC_SHADOW_BORINGSSL_SYMBOLS'
+    ss.dependency 'BoringSSL-GRPC', '0.0.6'
+    ss.compiler_flags = '-DBORINGSSL_PREFIX=GRPC'
```

Comment:
nit: why change that? is it purely cosmetic or is this required by something?

Reply:
`BORINGSSL_PREFIX` is the macro that BoringSSL uses to determine what prefix to use for its symbols.

Reply:
And by using BoringSSL's approach, we no longer need `GRPC_SHADOW_BORINGSSL_SYMBOLS` as that is gRPC's macro to enable symbol prefixing.